### PR TITLE
Livestream updates for Zoom configuration

### PR DESCRIPTION
**Pull request by aileenrose**
Much needed!

Screen Shot 2020-03-26 at 12 49 55 PM

### CODE REVIEW

File: handbook/marketing/livestream.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+ time to livestream | activity  
+ --------------- | ---------------    
+ 1 - 2 months out | Identify livestream information 
+ 1 month out | Send out invitations 
+ 2 weeks out | Send out invitation #2 
+ 1 day - 1 week out | Send out email reminders 
+ 1-2 weeks out | Perform a rehearsal 
+ 1 day out | Test livestream link 
+ 30 minutes | Final soundcheck 
+ Post livestream | Send out thank you emails with link to recorded livestream
```
Comment (creachadair) [inline handbook/marketing/livestream.md]: Minor formatting suggestion—to ensure this keeps readable as plain text.

```suggestion
 time to livestream | activity  
 ------------------ | ---------------    
 1 - 2 months out   | Identify livestream information 
 1 month out        | Send out invitations 
 2 weeks out        | Send out invitation #2 
 1 day - 1 week out | Send out email reminders 
 1-2 weeks out      | Perform a rehearsal 
 1 day out          | Test livestream link 
 30 minutes         | Final soundcheck 
 Post livestream    | Send out thank you emails with link to recorded livestream
```

File: handbook/marketing/livestream.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
+# Livestream events (Webinars)
+## Overview
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]:
```suggestion
# Livestream events (Webinars)

## Overview
```

File: handbook/marketing/livestream.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+The following information describes the planning and execution of a Sourcegraph livestream event. 
+
+## Livestream information
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]:
```suggestion
# Livestream events (Webinars)

The following information describes the planning and execution of a Sourcegraph livestream event. 

## Livestream information
```

File: handbook/marketing/livestream.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+- Short Description
+- Speaker Name
+- Speaker Bio
+- Speaker Headshot
+- Livestram Link from Zoom or GotoWebinar
+- Audience
+
+Sourcegraph uses these web applications for running a livestream event:
+
+- GotoWebinar or Zoom:
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]: We should settle on one or the other. I personally found GotoWebinar to not be a great experience.

File: handbook/marketing/livestream.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+ 2 weeks out | Send out invitation #2 
+ 1 day - 1 week out | Send out email reminders 
+ 1-2 weeks out | Perform a rehearsal 
+ 1 day out | Test livestream link 
+ 30 minutes | Final soundcheck 
+ Post livestream | Send out thank you emails with link to recorded livestream
+
+### Video conference setup
+
+Create the livestream, configuring the following:
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]:
```suggestion

### Creating and configuring a livestream

It's vitally important that only those authorized to contribute during the webinar are given the ability to share audio, video, and their screen, while participants can ask questions via chat only at this stage. These restrictions could be relaxed for events such as internal customer training.

Create the livestream with the following configuration:
```

File: handbook/marketing/livestream.md [hunk 6]
```diff
@@ -0,0 +1,117 @@
...
+
+- Set the schedule, host and details
+- Registration landing page form
+- Thanks for registering email which includes the livestream link
+
+Zoom required meeting configuration **IMPORTANT**
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]:
```suggestion
Zoom required meeting configuration
```

File: handbook/marketing/livestream.md [hunk 7]
```diff
@@ -0,0 +1,117 @@
...
+
+Zoom advanced configuration during broadcast, under 'Advanced Sharing Options'
+- Enable "One participant can share at a time"
+- Enable "Only Host" under "Who can share?"
+
+Zoom account management > account settings
+- Disable "File Transfer"
+- Disable "Allow Removed Participants to Rejoin"
+
+### Announcement on customer's Sourcegraph instance
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]: Does the announcement message need to include the password?

File: handbook/marketing/livestream.md [hunk 8]
```diff
@@ -0,0 +1,117 @@
...
+- Reminder email T-3 day which includes the Zoom livestream link
+- Reminder email T-1 day which includes the Zoom livestream link
+- Reminder email T-1 hour which includes the Zoom livestream link
+
+Since this is a [lead-generation activity](marketing_operations.md#maintaining-data-pipelines), we need to track it in our [HubSpot event log](https://docs.google.com/spreadsheets/d/16S3xlcY7DmpcfKZYD-3VHUsaPLiYHyisu8cD_gZpv0Q/edit?usp=drive_web&ouid=117507720010549543900). Add the HubSpot registration list to the `Livestream Registration` list and the attendees list to the `Livestream Attendee` list. Contact BizOps in the #marketing or #analytics channels. NOTE: This has not been implemented yet in HubSpot and Zapier yet.
+
+Post livestream, Sourcegraph will send followup emails:
+
+- Attendee thank you email with link to rewatch
+- Registered, but non-attendee email with link to watch
```
Comment (ryan-blunden) [inline handbook/marketing/livestream.md]:
```suggestion
## Post livestream

- A member of the marketing team will download, edit, then uploading the livestream to YouTube as an **unlisted** video
- Attendee thank you email with link to rewatch
- Registered, but non-attendee email with link to watch
```

File: handbook/marketing/livestream.md (hunk 4 above)
Comment (aileenrose) [reply handbook/marketing/livestream.md]: As we increase online events, GotoWebinar has the best integration with Hubspot.  With Zoom, a person cannot register for multiple events via Hubspot.  Zoom is okay if we are just planning one online meetup at a time.  Also, we have the option of having many to many meetup.    With more users on Zoom these days, maybe the integration with Hubspot will improve soon.

File: handbook/marketing/livestream.md (hunk 7 above)
Comment (aileenrose) [reply handbook/marketing/livestream.md]: no - it we will be included with the livestream link, emailed to the registrant.

**Pull request merged**